sdk/standbypool/azure-resourcemanager-standbypool/CHANGELOG.md

```diff
@@ -1,14 +1,125 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-09-25)
 
-### Features Added
+- Azure Resource Manager Standby Pool client library for Java. This package contains Microsoft Azure SDK for Standby Pool Management SDK. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+* `implementation.models.PagedOperation` was removed
+
+### Features Added
+
+* `implementation.models.StandbyContainerGroupPoolRuntimeViewResourceListResult` was added
```

Comment:
nit, we should exclude `implementation.models` (or anything under implementation namespace) from diffs https://github.com/Azure/azure-sdk-for-java/tree/main/eng/automation/changelog

Reply:
Yeah. 
Though the strange thing is that we actually didn't change this in this PR, but changelog reported it...

Reply:
This likely due to the sync process.

As we didn't pin the api-version, after they added the GA api-version, next SDK sync would already generate the code and updated the SDK.

So this PR likely mostly just version and samples (where they are not synced during the automation).

Reply:
Yeah, it is: https://github.com/Azure/azure-sdk-for-java/pull/41376/files#diff-acd7e9a2ae9298b55cdaf3e6a1d3b1799805bac8d6830e8d9ce5223d6182c7c3

Reply:
Ah, I see.

So not exactly about sync process, as usually it did not update commit ID.

It is due to a few non-typical sync that had to bump the commit ID (due to typespec behavior change etc.).

Reply:
> nit, we should exclude `implementation.models` (or anything under implementation namespace) from diffs https://github.com/Azure/azure-sdk-for-java/tree/main/eng/automation/changelog

issue created: https://github.com/Azure/azure-sdk-for-java/issues/42033